packages/gatsby/src/utils/feedback.ts

```diff
@@ -0,0 +1,74 @@
+import report from "gatsby-cli/lib/reporter"
+import { getConfigStore, getGatsbyVersion } from "gatsby-core-utils"
+import latestVersion from "latest-version"
+
+const feedbackKey = `feedback.disabled`
+const lastDateKey = `feedback.lastRequestDate`
+
+// This function is designed to be used by `gatsby feedback --disable`
+// and `gatsby feedback --enable`. This key is used to determine
+// if a user is allowed to be solicited for feedback
+export function setFeedbackDisabledValue(enabled: boolean): void {
+  getConfigStore().set(feedbackKey, enabled)
+}
+
+// Print the feedback request to the user
+export function showFeedbackRequest(): void {
+  getConfigStore().set(lastDateKey, Date.now())
```

Comment:
Maybe set this at the end in case they're offline & the version check doesn't finish?

Reply:
This doesn't do any network calls. It writes to disk!

Reply:
right but we don't want to set this unless we actually show them the prompt.

Reply:
Slightly confused! The prompt gets printed in the next two lines. This function only gets called if the user passes the heuristic check.

Reply:
oops, my bad — I was assuming there was one function which both tested against the heuristics & printed the message.